src/main/webapp/app/sentry/sentry.error-handler.ts

```diff
@@ -38,6 +38,11 @@ export class SentryErrorHandler implements ErrorHandler {
     }
 
     handleError(error: any): void {
+        // We ignore HttpError in the range 400-499
+        if (error.name === 'HttpErrorResponse' && error.status < 500 && error.status >= 400) {
+            return;
```

Comment:
should we still throw the error to the console?

Reply:
Good one, fixed